R/bind_ard.R

```diff
@@ -18,7 +22,7 @@
 #' ard <- ard_categorical(ADSL, by = "ARM", variables = "AGEGR1")
 #'
 #' bind_ard(ard, ard, .update = TRUE)
-bind_ard <- function(..., .update = FALSE) {
+bind_ard <- function(..., .update = FALSE, .order = TRUE) {
```

Comment:
This function is used in the construction of other `ard_*()` functions, so can we make the default `.order = FALSE` so there is no unexpected sorting? This will also mirror the behavior of `dplyr::bind_rows()` more similarly as well.